scc: add page (#4248)
tldr-pages/tldr: pages/common/scc.md

# scc

> Tool written in Go that counts lines of code.
> More information: <https://github.com/boyter/scc>.

- Print lines of code in the current directory:

`scc`

- Print lines of code in the target directory:

`scc {{path/to/directory}}`

- Display output for every file:

`scc --by-file`

> - Display output using specific output format, default is tabular:

Reviewer: Perhaps this will read better?
```suggestion
- Display output using a specific output format (defaults to `tabular`):
```

`scc --format {{tabular|wide|json|csv|cloc-yaml|html|html-table}}`

> - Display output limit to file extension:

Reviewer: Perhaps this will read better?
```suggestion
- Only count files with specific file extensions:
```

`scc --include-ext {{go, java, js}}`

> - Display output exclude directories:

Reviewer: Perhaps this will read better?
```suggestion
- Exclude directories from being counted:
```

`scc --exclude-dir {{.git,.hg}}`

> - Display output and sort by column, default by files:

Reviewer: Perhaps this will read better?
```suggestion
- Display output and sort by column (defaults to by files):
```
